src/Benchmarks/Program.cs

```diff
@@ -186,5 +187,21 @@ private static void Listen(KestrelServerOptions options, IConfigurationRoot conf
                 }
             });
         }
+
+        private static IPEndPoint CreateIPEndPoint(Uri uri)
+        {
+            IPAddress ip;
+
+            if (string.Equals(uri.Host, "localhost", StringComparison.OrdinalIgnoreCase))
+            {
+                ip = IPAddress.Loopback;
```

Comment:
What you have now should be sufficient to unblock most scenarios.  We should revisit this after the final design though.